lib/dependabot/file_updaters/java/maven.rb

```diff
@@ -9,7 +9,8 @@ module FileUpdaters
     module Java
       class Maven < Dependabot::FileUpdaters::Base
         DECLARATION_REGEX =
-          %r{<dependency>.*?</dependency>|<plugin>.*?</plugin>}m
+          %r{<parent>.*?</parent>|<dependency>.*?</dependency>
+             |<plugin>.*?</plugin>}mx
```

Comment:
This regex is also split over multiple line, but the `x` flag lets the tests pass.

Reply:
I'd move the `|` to the end of the previous line.